src/juicebox/tools/clt/old/PreProcessing.java

```diff
@@ -59,6 +59,8 @@ public PreProcessing() {
                 + "           : -n Don't normalize the matrices\n"
                 + "           : -z <double> scale factor for hic file\n"
                 + "           : -a <1, 2, 3> filter based on inner, outer, or tandem alignment\n"
+                + "           : --randomize_pos randomize positions based on seed\n"
```

Comment:
Can you set the parameter --randomize-pos-seed? Use hyphens not underscore

Reply:
or just --seed tbh?